### ISSUE TITLE
Support spread operators

### ISSUE THREAD

**Issue by kafura** [vuedoc/parser#22]
If I use spread operators like this:
 ``` javascript
computed: {
  ...mapGetters('map', [
    'searchMapToolIsActive'
  ])
}
```
I get the following error:
`Uncaught TypeError: Cannot read property 'type' of undefined`

The `property.key` property is undefined, see this image:
![image](https://user-images.githubusercontent.com/566659/37388385-8c92e250-2760-11e8-9084-695837ff94c5.png)

**Comment by demsking** [comment]
Sorry, but the spread operators is not yet supported. I'll implement it soon

**Comment by demsking** [comment]
Implemented in c08f0d6. It'll be released soon

**Comment by demsking** [comment]
**This project has been moved to https://gitlab.com/vuedoc/parser**
